docs/permissions/getting-started.md

```diff
@@ -34,129 +22,48 @@ Like many other parts of Backstage, the permissions framework relies on informat
 
 [The IdentityResolver docs](../auth/identity-resolver.md) describe the process for resolving group membership on sign in.
 
-## Optionally add cookie-based authentication
-
-Asset requests initiated by the browser will not include a token in the `Authorization` header. If these requests check authorization through the permission framework, as done in plugins like TechDocs, then you'll need to set up cookie-based authentication. Refer to the ["Authenticate API requests"](https://github.com/backstage/backstage/blob/master/contrib/docs/tutorials/authenticate-api-requests.md) tutorial for a demonstration on how to implement this behavior.
-
 ## Integrating the permission framework with your Backstage instance
 
+:::info
+If you created your backstage app using the [@backstage/create-app](https://backstage.io/docs/getting-started/#1-create-your-backstage-app), the permission framework will already be setup including the allow-all policy!
+:::
```

Comment:
Ah I forgot to delete the earlier comment after suggesting the concrete update 😅. I'm a lil bit worried that this will not have people skip step 2, which is still important to enable permissions. As a side note I'm thinking we should change the default there too, but let's consider that separately.